docs/reference/runtime/Execution.md

```diff
@@ -1,5 +1,9 @@
 # Execution
 
+Embedded systems are systems that are built for performing a sole dedicated duty. They are typically constrained in resources and contain only the necessary components to perform their duty. Most embedded systems are powered by a single microcontroller chip. That microcontroller is typically also limited in resources such as memory. Software for an embedded system must be carefully designed to make the best use of such limited resources. Mbed OS is often too large for such systems in its default configuration, but support a bare metal mode that might be a better option depending on your target system.
```

Comment:
```suggestion
Embedded systems are dedicated to performing a single duty. They are typically constrained in resources and contain only the necessary components to perform that duty. Most embedded systems are powered by a single microcontroller chip, which is typically limited in resources such as memory. Software for an embedded system must be carefully designed to make the best use of these limited resources. Mbed OS in its default configuration is often too large for constrained systems, but supports a bare metal mode that might be a better option for your target system.
```

Reply:
- I'm not sure what the relationship between the two paragraphs is. The content about constrained systems isn't related to the Threading, and the idea that some systems can't handle Threading is mentioned in both paragraphs.
- Bare metal is explained in detail in its own book, and in the general intro to Mbed OS.

In short, I think you don't need the first paragraph.